integrations/chroma/src/haystack_integrations/document_stores/chroma/document_store.py

```diff
@@ -116,7 +117,8 @@ def _ensure_initialized(self):
             if "hnsw:space" not in self._metadata:
                 self._metadata["hnsw:space"] = self._distance_function
 
-            if self._collection_name in client.list_collections():
+            existing_collection_names = [c.name for c in client.list_collections()]
+            if self._collection_name in existing_collection_names:
```

Comment:
These were the lines that needed to be changed to support chroma v1.0.0

Reply:
Interesting https://cookbook.chromadb.dev/core/collections/#deleting-a-collection :

> In ChromaDB 0.6.0, the list_collections method was updated to return a list of collection names (Sequence[CollectionName] where CollectionName is a string) instead of collection objects.

Seems like now they reverted that decision:
https://github.com/chroma-core/chroma/commit/069ca5fb5223a1e466244f3c6e858c8e183ce233